.github/workflows/deploy-docs.yaml

```
@@ -38,4 +38,9 @@ jobs:
         uses: autowarefoundation/autoware-github-actions/deploy-docs@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          latest: ${{ github.ref_name == github.event.repository.default_branch }}
+          latest: ${{ github.event.workflow_run.event != 'pull_request_target' && github.ref_name == github.event.repository.default_branch }}
```

```suggestion
          latest: ${{ github.event_name != 'pull_request_target' && github.ref_name == github.event.repository.default_branch }}
```

https://docs.github.com/en/actions/learn-github-actions/contexts#example-usage-of-the-github-context